templates/tfengine/recipes/resources.hcl

```diff
@@ -648,7 +690,21 @@ schema = {
             type        = "string"
           }
           network_project_id = {
```

Comment:
is there way to just define common attributes for different types of resources just once?

Reply:
I can't think of a good way. Maybe we can add reference type of support so we don't need to redefine the fields and can just say this was already defined in an upper layer?

Reply:
Yea, maybe not in an upper layer. I was thinking if it is possible to do it in a way that does not affect the end user config, like users will not have another nested block for network_project_id and network_project_suffix and other common stuff, but just reduce attribute definition duplications in the schema. We can leave it as is, I was just wondering if that's even possible.

Reply:
Actually I think it is possible. We could define this kinda block at the top level data block, and then just make sure we do something like

network_project := {{get . "network_project_suffix" (get $ "network_project_suffix)}}